projects/chaos/inc/sequencer.h

```diff
@@ -30,6 +31,8 @@ typedef struct SequencerStorage {
   const SequencerEventPair *events_it;
   const SequencerEventPair *events_end;
   bool awaiting_response;
+  bool started;
```

Comment:
Are these 3 bools all independent of each other? Is it possible to be started and stopped at the same time?

Reply:
Yes:
- `awaiting_response` indicates we are awaiting a response event
- `started` is set once we see the first message in the sequence (used for flushing.
- `stopped` indicates we should stop raising events after we see the one we are waiting for.

Perhaps renaming `s/stopped/end_after_await/` is less confusing?

Reply:
Yeah, maybe `end/stop_after_response`?

Reply:
Done.